### DISCUSSION TITLE
accommodationFloorPlan

### DISCUSSION THREAD

**Discussion by tranlongphu** [RehanSaeed/Schema.NET#152] [Q&A]
IHi, 'm looking for FloorPlan under Residence but do not see it, is it in the latest version ? Thanks. 
https://pending.schema.org/accommodationFloorPlan

**Comment by RehanSaeed** [comment]
This type is `pending` in schema.org:

>  This term is proposed for full integration into Schema.org, pending implementation feedback and adoption from applications and websites.

It won't make it until it is properly released. In the mean time, you can create your own type inheriting from our types.
